tools/translate_japanese_posts.sh

```
@@ -0,0 +1,309 @@
+
+# Function to find posts with Japanese translations
+find_japanese_posts() {
+    print_status "Scanning for posts with Japanese translations..."
+    
+    local japanese_posts=()
+    
+    # Find all directories in the post directory
+    while IFS= read -r -d '' post_dir; do
+        local japanese_file="$post_dir/index.ja.md"
+        if [[ -f "$japanese_file" ]]; then
+            local post_name=$(basename "$post_dir")
```

Shellcheck (0.10.0)

[warning] Declare and assign separately to avoid masking return values.

(SC2155)